lace/archive/gadget_archive.py

```diff
@@ -299,7 +305,14 @@ def _get_nz_linP(self, sim_label):
             linP_zs = fit_linP.get_linP_Mpc_zs(sim_cosmo, zs, self.kp_Mpc)
             linP_zs = list(linP_zs)
 
-        return zs, linP_zs
+            labels = ["z", "Delta2_p", "n_p", "alpha_p", "f_p"]
+            emu_cosmo = {}
+            for lab in labels:
+                emu_cosmo[lab] = np.zeros(zs.shape[0])
+                for ii in range(zs.shape[0]):
+                    emu_cosmo[lab][ii] = linP_zs[ii][lab]
+
+        return cosmo, emu_cosmo
```

Comment:
The docstring says you are returning zs, linP params. I guess it is not updated? 

Here you are returning the configuration dictionary of Gadget, and a list of redshifts and emu params, right?

Reply:
Yes, sorry, I forgot about all docstrings. I will update them on Monday morning and work on fixing all problems